src/DynamoCore/Models/DynamoModel.cs

```diff
@@ -212,8 +212,8 @@ public string Version
         /// <summary>
         /// Name of the Host (i.e. DynamoRevit/DynamoStudio)
         /// </summary>
-        [Obsolete("This property will be removed in Dynamo 3.0 - please use HostAnalyticsInfo")]
-        public string HostName { get; set; }
+        [Obsolete("This property will be removed in a future version of Dynamo - please use HostAnalyticsInfo")]
+        internal string HostName { get; set; }
```

Comment:
is this used in a lot of places?

Reply:
10 places. But there are some comments in the code that indicates that HostAnalyticsInfo is missing at startup in some cases. I don't think we want to deal with that right now.

Reply:
I think there are PRs from both @BogdanZavu and @zeusongit that also conflict here so lets sort that out after this goes in.